packages/controller/io-package.json

```diff
@@ -539,6 +539,35 @@
         "object": 1604
       }
     },
+    {
+      "_id": "system.licenses",
+      "type": "config",
+      "common": {
+        "name": {
+          "en": "Licenses from iobroker.net",
+          "de": "Lizenzen von iobroker.net",
+          "ru": "Лицензии от iobroker.net",
+          "pt": "Licenças de iobroker.net",
+          "nl": "Licenties van iobroker.net",
+          "fr": "Licences de iobroker.net",
+          "it": "Licenze da iobroker.net",
+          "es": "Licencias de iobroker.net",
+          "pl": "Licencje z iobroker.net",
+          "zh-cn": "来自 iobroker.net 的许可证"
+        }
+      },
+      "native": {
+        "login": "",
+        "password": "",
```

Comment:
not better an authenticated jwt or such instead of password?

Reply:
Better is oauth2 but cloud does not have yet this functionality